src/Spout/Writer/XLSX/Helper/StyleHelper.php

```diff
@@ -71,6 +82,32 @@ protected function registerFill($style)
         }
     }
 
+    /**
+     * Register a border definition
+     *
+     * @param $style
+     */
+    protected function registerBorder($style)
+    {
+        $styleId = $style->getId();
+
+        if (true === $style->shouldApplyBorder()) {
+
+            $border = $style->getBorder();
+            $serializedBorder = serialize($border);
+
+            if (!isset($this->registeredBorders[$serializedBorder])) {
+
+                $this->registeredBorders[$serializedBorder] = $styleId;
+                $this->styleIdToBorderMappingTable[$styleId] = count($this->registeredBorders);
+
+            }
+
+        } else {
+            $this->styleIdToBorderMappingTable[$styleId] = 0;
```

Comment:
Damn, I actually read it wrong. I thought the else was following the `if (!isset($this->registeredBorders[$serializedBorder]))` statement. The 0 makes total sense :)